client/app/kubernetes/containers/kubernetes-containers.html

```diff
@@ -0,0 +1,25 @@
+<div>
+    <h2 class="oui-header_2" data-translate="kube_containers_coming_soon"></h2>
+    <span data-translate="kube_containers_description"></span>
+    <div>
```

Comment:
You can get rid of this `<div>` and for instance apply `d-block` helper class on your `<span>` element (or use a regular `<p>` element).